vendor/modules.txt

```diff
@@ -100,6 +100,7 @@ github.com/dgrijalva/jwt-go
 # github.com/dimchansky/utfbom v1.1.0
 github.com/dimchansky/utfbom
 # github.com/getkin/kin-openapi v0.13.0
+## explicit
```

Comment:
What does this do?

Reply:
I'm not sure. This is automatically generated but it is all around this file. Maybe another legacy stuff cause by Go import path vs. Go modules? https://golang.org/cmd/go/#hdr-Module_compatibility_and_semantic_versioning